site-manage/site-group-manager/src/main/resources/Messages_eu.properties

```diff
@@ -0,0 +1,172 @@
+# Tool
```

Comment:
Looks like this tool is not in EU, which is a problem, could you confirm @pablosanroman ?

@V100X Could you only put our strings and no the whole file?